content/sdk/00.index.md

```diff
@@ -1,21 +1,147 @@
 ---
-title: zkSync Era SDKs
-description:
+title: "Home"
+menuTitle: "Introduction"
+description: "zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network."
+navigation: true
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript, go, swift, python, java, rust"
 ---
 
-zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network,
-a high-performance scaling solution for Ethereum.
+## What is zkSync Era SDKs?
+
+zkSync Era SDKs help developers create decentralized applications (dApps) for the zkSync Era network. zkSync Era is a
+Layer 2 solution for Ethereum. It uses ZK rollup technology to provide scalable and low-cost transactions, keeping the
+security and trust of the Ethereum blockchain.
+
+<div align="center">
+    <img width="50%" src="/images/sdk/zksync-era-sdks.png" alt="zkSync Era SDKs"/>
+</div>
```

Comment:
We can keep this in here for now but we should change this later to use the markdown image syntax.
```
::div{class="text-center"}
![zkSync Era SDKs](/images/sdk/zksync-era-sdks.png){ width="50%"}
::
```

Right now the markdown image component has limited styling options which I'll have to edit for it to work.